### DISCUSSION TITLE
what happened

### DISCUSSION THREAD

**Discussion by DebaltsevK0t3l** [JargeZ/vhs#59] [Q&A]
i need a help, l mac os user and according to the instructions, I installed the application, and when I started rendering the video, an error occurs with breaking the application. 

![ 2022-08-27 в 19 05 10](https://user-images.githubusercontent.com/112267643/187038423-d1e8f139-4b07-467c-abda-32468bab6016.png)

2022-08-27 20:10:51.894 | ERROR    | __main__:crash_handler:20 - Uncaught exception: 'NtscApp' object has no attribute 'ProcessAudioCheckBox'
NoneType: None

**Comment by SergeyMC9730** [comment]
#57 

We have one more record of this bug.
Maybe, I can try to look up, what causes this issue. I will tell you, when I get any ideas what is wrong

**Reply by DebaltsevK0t3l** [reply]
> #57
> 
> We have one more record of this bug. Maybe, I can try to look up, what causes this issue. I will tell you, when I get any ideas what is wrong

Perhaps you have the 'ProcessAudioCheckBox' attribute missing in the 'NtscApp' class for mac OS. The render block accesses the attribute, but due to the absence of such and such, it refuses to work.

**Reply by SergeyMC9730** [reply]
Made a Pull Request #65
